lib/fs.js

```diff
@@ -1984,6 +1984,7 @@ ReadStream.prototype.open = function() {
 
     self.fd = fd;
     self.emit('open', fd);
+    self.emit('ready', fd);
```

Comment:
I would omit `fd` here, the other streams don’t have this on the event (and it doesn’t make sense to have it for `Http2Stream`)

Reply:
I have removed the`fd` for the ready event.